src/main/java/gift/global/security/AuthFilter.java

```diff
@@ -9,18 +9,18 @@
 import org.springframework.web.filter.OncePerRequestFilter;
 
 @Component
-public class JwtAuthFilter extends OncePerRequestFilter {
+public class AuthFilter extends OncePerRequestFilter {
 
     private final JwtProvider jwtProvider;
 
-    public JwtAuthFilter(JwtProvider jwtProvider) {
+    public AuthFilter(JwtProvider jwtProvider) {
         this.jwtProvider = jwtProvider;
     }
 
     @Override
     protected boolean shouldNotFilter(HttpServletRequest request) throws ServletException {
         String path = request.getRequestURI();
-        return !path.startsWith("/api/wishes"); // "/api/wishes"로 시작하지 않으면 필터 적용하지 않음
+        return !path.startsWith("/api/wishes") && !path.startsWith("/api/orders");
```

Comment:
```suggestion
        // 클래스 상수로 관리
        List<String> excludedPrefixes = List.of("/api/wishes", "/api/orders");
        
        ...
        
        return excludedPrefixes.stream().noneMatch(path::startsWith);
```
이런식으로 관리하면 적용할 path가 변경 되었을 때 로직 변경을 하지 않아도 되어 filter 대상의 쉬운 추가 삭제가 가능하고 안정성이 올라갑니다!